src/proto/generation.proto

```diff
@@ -325,9 +413,12 @@ message Request {
     ImageParameters image = 5;
     ClassifierParameters classifier = 7;
     AssetParameters asset = 8;
+    InterpolateParameters interpolate = 11;
+    TransformParameters transform = 12;
   }
   optional ConditionerParameters conditioner = 6;
   reserved 9,10; //9, 10 are reserved
+  optional google.protobuf.Struct extras = 2047; // for development use
```

Comment:
👀 ?

Reply:
This speeds up development by allowing passing essentially arbitrary JSON from client to server without having to branch and update api-interfaces. So you can test out new parameters and new messages in development then promote to actual message in the proto when ready and before going to main. It is currently used by warp_flow which has a temporary format for dev and testing (until further project+asset work).

Reply:
Ah - good 👍 . I've wanted this all along too.